sdk/src/nonce_state.rs

```diff
@@ -237,21 +259,6 @@ mod test {
         })
     }
 
-    #[test]
-    fn nonce_inx_uninitialized_account_not_signer_fail() {
-        let rent = Rent {
-            lamports_per_byte_year: 42,
-            ..Rent::default()
-        };
-        let min_lamports = rent.minimum_balance(NonceState::size());
-        with_test_keyed_account(min_lamports + 42, false, |nonce_account| {
-            let signers = HashSet::new();
-            let recent_blockhashes = create_test_recent_blockhashes(0);
-            let result = nonce_account.nonce(&recent_blockhashes, &rent, &signers);
-            assert_eq!(result, Err(InstructionError::MissingRequiredSignature),);
-        })
-    }
-
     #[test]
     fn nonce_inx_initialized_account_not_signer_fail() {
```

Comment:
Not directly related to this PR but what does `inx` mean?

Reply:
"instruction" is a lot of letters.  I thought better to save them for describing the test